### PR TITLE
Add ProviderDomain field and send acceptForward request

### PR DESCRIPTION
**Pull request by ishank011**

### CODE REVIEW

File: ui/components/AcceptInvite.vue [hunk 1]
```diff
@@ -30,20 +32,23 @@ export default {
...
-          title: this.$gettext("This feature is not yet implemented")
-        });
-      }
-    }
-  },
-  watch: {
-    inputValue() {
-      this.$emit("input", this.inputValue);
+    onConfirm(tokenInput, providerInput) {
+      let currentObj = this;
```
Comment (LovisaLugnegard) [inline ui/components/AcceptInvite.vue:36]: I think we should rename `currentObject` to be somewhat more descriptive.

Comment (ishank011) [reply ui/components/AcceptInvite.vue:36]: I just used it to return the current context. Any suggestions for the rename?

File: ui/components/AcceptInvite.vue [hunk 2]
```diff
@@ -15,11 +15,13 @@
...
 
 <script>
+import axios from 'axios';
+
 export default {
   name: "AcceptInvite",
   data: function() {
     return {
-      inputValue: "Token"
+      localhost: "http://localhost:19001"
```
Comment (LovisaLugnegard) [inline ui/components/AcceptInvite.vue:24]: I think this should be added here to get away from some errors:
```
      tokenInput: null,
      providerInput: null,
```

File: ui/components/AcceptInvite.vue [hunk 3]
```diff
@@ -30,20 +32,23 @@ export default {
...
-        this.showMessage({
-          title: this.$gettext("This feature is not yet implemented")
-        });
-      }
-    }
-  },
-  watch: {
-    inputValue() {
-      this.$emit("input", this.inputValue);
+    onConfirm(tokenInput, providerInput) {
```
Comment (LovisaLugnegard) [inline ui/components/AcceptInvite.vue:35]: I think we should check so there are values provided for `tokenInput` and `providerInput` before proceeding here.

File: ui/components/AcceptInvite.vue [hunk 4]
```diff
@@ -30,20 +32,23 @@ export default {
...
+      })
+      .then(function (response) {
+        currentObj.output = response.data;
+      })
+      .catch(function (error) {
+        currentObj.output = error;
+      });
+
+      this.showMessage({
+        title: this.$gettext("Invite accepted.")
```
Comment (LovisaLugnegard) [inline ui/components/AcceptInvite.vue:50]: "Invite accepted" shouldn't be shown if there was an error

File: ui/components/AcceptInvite.vue [hunk 5]
```diff
@@ -30,20 +32,23 @@ export default {
...
-  },
-  watch: {
-    inputValue() {
-      this.$emit("input", this.inputValue);
+    onConfirm(tokenInput, providerInput) {
+      let currentObj = this;
+
+      axios.post(this.localhost + '/ocm/invites/forward/', {
+        token: tokenInput,
+        providerDomain: providerInput
```
Comment (LovisaLugnegard) [inline ui/components/AcceptInvite.vue:40]: I guess we need to add some auth info here?

File: ui/components/AcceptInvite.vue (hunk 1 above)
Comment (LovisaLugnegard) [reply ui/components/AcceptInvite.vue:36]: Do we need to store this, or could we just return `response.data` and `error`? Do we need to return anything? Or if we need to and this is the current context, why not call it `context`? 
Sry for all the questions.

Comment (ishank011) [reply ui/components/AcceptInvite.vue]: Removed it altogether

File: ui/components/AcceptInvite.vue (hunk 2 above)
Comment (ishank011) [reply ui/components/AcceptInvite.vue]: These are parameters passed to the onConfirm method, so they don't exist outside its scope.

File: ui/components/AcceptInvite.vue (hunk 4 above)
Comment (ishank011) [reply ui/components/AcceptInvite.vue]: Done

File: ui/components/AcceptInvite.vue (hunk 3 above)
Comment (ishank011) [reply ui/components/AcceptInvite.vue:34]: Done

File: ui/components/AcceptInvite.vue [hunk 6]
```diff
@@ -30,21 +31,33 @@ export default {
...
+
+      if (tokenInput !== "" && providerInput !== "") {
+        const data = {
+          'tokenInput': tokenInput,
+          'providerInput': providerInput
+        };
+        const params = {
+          method: 'POST',
+          mode: 'cors',
+          body: JSON.stringify(data)
```
Comment (LovisaLugnegard) [inline ui/components/AcceptInvite.vue:44]: Is there a specific reason for not sending the body as JSON?

**Pull request closed without merging**